### PR TITLE
feat(thumbails-illustration): add trendline

### PR DESCRIPTION
**Pull request by kagnamazian**
## What to test? Scope of the test?
Add thumbnail illustration for trendline

## Reviews
Reviewers: @chiel @MLKiiwy @kagnamazian @realdreamer @ranst91-u

## Previews : 
image

### CODE REVIEW

File: src/assets/media/thumbnail_trendline.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <path d="M-96 65.022c0 12.804 39.557 47.487 77.785 35.148 38.228-12.339 47.228-32.21 72.228-47.148 25-14.939 73 54.863 102 27.431 29-27.431 27-57.335 42-50.536 15 6.8 82.516 81.714 108.793 70.253C333.083 88.709 352.708-96.796 388.513-97c35.805-.203 71.9 103.983 118.55 111.797 46.65 7.813 69.45-61.402 94.45-40.088 25 21.313 75.125 118.06 134.39 90.313 59.266-27.747 79.438-84.749 89.524-83" stroke="#00A5C9" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" mask="url(#b)"/>
+    <circle stroke="#00A5C9" stroke-width="2" fill="#FFF" mask="url(#b)" cx="61" cy="51" r="4"/>
+    <circle stroke="#00A5C9" stroke-width="2" fill="#FFF" mask="url(#b)" cx="141" cy="87" r="4"/>
+  </g>
+</svg>
```
Comment (chiel) [inline src/assets/media/thumbnail_trendline.svg]: Please add a new line at the end of the file :)

File: src/assets/media/thumbnail_trendline.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<svg width="179" height="127" viewBox="0 0 179 127" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (ranst91-u) [inline src/assets/media/thumbnail_trendline.svg]: LGTM! 
Just one small thing:
There are several elements that can be removed (including xmlns). They are just referencing each other, and SVGOMG is probably not smart enough to make the necessary changes.

Comment (kagnamazian) [reply src/assets/media/thumbnail_trendline.svg]: I left exactly the same thing as the other asset we have in stylable. The xmlns is present there as well. 
Do you have another example of things I should remove and SVGOMG didn't see it?

Comment (chiel) [reply src/assets/media/thumbnail_trendline.svg]: Hmm, `xmlns` was required I thought? And `xlink:href` is used for example, which makes `xmlns:xlink` necessary? Do you have any reading material on this? Would be interested in learning more :)

Comment (ranst91-u) [reply src/assets/media/thumbnail_trendline.svg]: tbh, I managed to remove them just by experimenting.
I will grab the svg, do the change (manually) and post it

**Pull request merged**